docs/tutorials/ifci_px4_flight_controller.rst

```diff
@@ -0,0 +1,320 @@
+
+Setting Up PX4 for IQUART and Building
+======================================
+
+Once the toolchain is set up, you must change the settings for your board to turn on Vertiq IQUART integrations. To do this, enter your PX4 directory and use the command below, but replace ``<your-flight-control-board>`` with your flight control board's name. Additionally a command with the example board we are using is shown below for reference.
+
+.. code-block:: bash
+
+    make <your-flight-control-board> boardconfig
+
+.. code-block:: bash
+
+    make px4_fmu-v6c boardconfig
+
+This should bring up the boardconfig window in the terminal. The 'Model' section should have your board name. In the image below we are using an ``fmu-v6c``.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/board_config_main.png
+    :align: center
+    :scale: 50
+    :alt: Main Board Config Page
+
+    Main Board Config Page
+
+Navigate to the ``drivers`` subsection with the arrow keys and press ``Enter`` to enter it.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/board_config_select_drivers.png
+    :align: center
+    :scale: 50
+    :alt: Selecting the Drivers Section
+
+    Selecting the Drivers Section
+
+Navigate to the ``actuators`` submenu with the arrow keys and press ``Enter`` to enter it.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/board_config_select_actuators.png
+    :align: center
+    :scale: 50
+    :alt: Selecting the Actuators Section
+
+    Selecting the Actuators Section
+
+Navigate to the ``vertiq_io`` subsection and press space to select it. An asterisk should appear in the box. Once the asterisk is in the box, press ``Enter`` to enter the vertiq_io submenu.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/actuators_select_vertiq_io.png
+    :align: center
+    :scale: 50
+    :alt: Selecting the vertiq_io Section
+
+    Selecting the vertiq_io Section
+
+Inside the ``vertiq_io`` submenu one option should appear for including IFCI configuration parameters. Select this by pressing space.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/ifci_unselect.png
+    :align: center
+    :scale: 50
+    :alt: Include IFCI Parameters
+
+    Include IFCI Parameters
+
+Once this is selected, a second option will appear for including pulsing module configurations. If you plan on using `underactuated propellers <https://www.vertiq.co/up12>`_, select this as well.
+
+Once done, press ``Q`` and then ``Y`` to save the configuration.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/save.png
+    :align: center
+    :scale: 50
+    :alt: Save the Configuration
+
+    Save the Configuration
+
+Once saved, build the firmware with the following command, replacing ``your-flight-control-board`` with the name of your flight control board. This will be the same name as used in the :ref:`previous steps<Setting Up the PX4 Toolchain>`.
+
+.. code-block:: bash
```

Comment:
can you add the same thing we did above for the boardconfig? Just add a real example for the board you're using. I know it's redundant...but I'd rather be redundant than have people yell at us because they can't figure anything out.